code/src/ErrorMessage.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export const ErrorMessage = () => {
+    return (
+        <div className='error'>
+            <span>The best kind of happy is within 5 to 140 characters! <span role="img" aria-label="confetti">🎉</span></span>
```

Comment:
Here you use both `'` and `"`. It would be nice to choose one way of coding instead of mixing